src/server/models/Player.js

```diff
@@ -101,6 +101,12 @@ export default class Player extends Model {
 
       handled = await reaction.reactTo(event, this, player, ...args) || handled;
     }
+
+    const durationReactions = this.playArea.filter(card => card.shouldReactTo(event, this, player, ...args));
```

Comment:
hmm, I might have to change this later....
especially since lighthouse isn't actually a Reaction